src/code_manager/model/CssGenerator.js

```diff
@@ -121,8 +125,10 @@ export default Backbone.Model.extend({
       }
     });
 
-    if ((selectorStrNoAdd && found) || selectorsAdd || singleAtRule) {
-      const block = rule.getDeclaration();
+    if ((selectorStrNoAdd && found) || selectorsAdd || singleAtRule || rule.attributes.style) {
```

Comment:
`rule.attributes.style` is an object, it's a truthy value, so the condition is pointless